FLIR/conservator/util.py

```diff
@@ -78,6 +78,16 @@ def download_file(path, name, url, silent=False, no_meter=False):
     return True
 
 
+def md5sum_file(path, block_size=1024 * 1024):
+    hasher = hashlib.md5()
+    with open(path, "rb") as fp:
+        block = fp.read(block_size)
+        while block:
+            hasher.update(block)
+            block = fp.read(block_size)
+    return hasher.hexdigest()
```

Comment:
IIRC, `LocalDataset` also gets MD5 of local files when uploading staged images. Can you update that to use this utility?

Reply:
line 300

Reply:
Good idea. Folder download is the other place I'm planning to use it.